### PR TITLE
Add Changes from Patch

### PR DESCRIPTION
**Pull request by CrystalJewell**
Updated `index.js` and `chat.js` with changes made via patch by @grossvogel  in another project.

### CODE REVIEW

**Comment by CrystalJewell** [comment]
@grossvogel Whenever you have some spare time, if you could look this over to make sure I got all the changes pulled in and in the correct place.  There is one change I had to make with `defaultProps.customerData` to remove identifying info. There may have to be a check to see if we have a `name` and `email` and if not just pass in what we want for default into the props instead of having it hardcoded here.

File: LiveChat/components/Chat.js [hunk 1]
```diff
@@ -48,15 +48,22 @@ export default class Chat extends React.Component {
...
 					ref={(ref) => {
 						this.chat = ref
 					}}
 				>
-					<NavigationBar chatTitle={chatTitle} closeChat={closeChat} />
-					{isReconnecting && <Text style={styles.connectionStatus}>Reconnecting...</Text>}
-					{headerText && <Text style={styles.status}>{headerText}</Text>}
+					{this.props.NavBarComponent ? (
+						<this.props.NavBarComponent chatTitle={chatTitle} closeChat={closeChat} isReconnecting={isReconnecting}
+																				headerText={headerText} />
```
Comment (grossvogel) [inline LiveChat/components/Chat.js]: Formatting ended up a bit strange here, but not a big deal

**Review by grossvogel — APPROVED** [review]
Nice, thank you! I don't think we need to worry about the `customerData` prop default because we'll always be supplying a specific value for that prop.

File: LiveChat/components/Chat.js (hunk 1 above)
Comment (CrystalJewell) [reply LiveChat/components/Chat.js]: It did, I used the built in formatted, I'm not sure why it decided it needed to move that last one so far out though.  I made a fix though, da00882

**Pull request merged**